src/main/java/toughasnails/api/season/SeasonHelper.java

```diff
@@ -55,4 +57,15 @@ public static boolean canSnowAtTempInSeason(Season season, float temperature)
         ISeasonData getServerSeasonData(World world);
         ISeasonData getClientSeasonData();
     }
+
+	public static float getSeasonFloatTemperature(Biome biome, BlockPos pos, Season season) {
+        if (biome.getTemperature() <= 0.7F && season == Season.WINTER && SyncedConfig.getBooleanValue(SeasonsOption.ENABLE_SEASONS))
+        {
+            return 0.0F;
+        }
+        else
+        {
+            return biome.getFloatTemperature(pos);
+        }
+	}
```

Comment:
We replace tabs with 4 spaces, simply because it ensures the code displays consistently in different web browsers, and it's pretty well supported by most IDEs as well.